liquibase-standard/src/main/java/liquibase/changelog/ChangeSet.java

```diff
@@ -1644,23 +1644,32 @@ private String addSqlMdc(Change change, Database database, boolean generateRollb
         // If the change is for this Database
         // add a Boolean flag to Scope to indicate that the Change should not be executed when adding MDC context
         //
-        if (! change.supports(database)) {
+        if (!change.supports(database)) {
             return null;
         }
+
         AtomicReference<SqlStatement[]> statementsReference = new AtomicReference<>();
         Map<String, Object> scopeValues = new HashMap<>();
         scopeValues.put(Change.SHOULD_EXECUTE, Boolean.FALSE);
-        StringBuilder sqlStatementsMdc = new StringBuilder();
+        StringBuilder commandsMdc = new StringBuilder();
+
         Scope.child(scopeValues, () -> {
-            statementsReference.set(generateRollbackStatements ?
-                    change.generateRollbackStatements(database) : change.generateStatements(database));
-            sqlStatementsMdc.append(Arrays.stream(statementsReference.get())
-                    .map(statement -> SqlUtil.getSqlString(statement, SqlGeneratorFactory.getInstance(), database))
-                    .collect(Collectors.joining("\n")));
+            SqlStatement[] statements = generateRollbackStatements ? change.generateRollbackStatements(database) : change.generateStatements(database);
+
+            statementsReference.set(statements);
+
+            String formattedCommands = Arrays.stream(statementsReference.get())
+                    .map(database::formatCommandForMdc)
+                    .filter(Objects::nonNull)
+                    .collect(Collectors.joining("\n"));
+
+            commandsMdc.append(formattedCommands);
```

Comment:
Suggest you change the word `commands` here to `statements` or `sqlStatements`, like it was prior to your changes.

Reply:
Here it could stay `statementsMdc` , just getting rid of the `sql` word.

Reply:
done